src/NetAnalyzers/UnitTests/Microsoft.NetCore.Analyzers/Publish/AvoidAssemblyLocationInSingleFileTests.cs

```diff
@@ -8,7 +8,6 @@
     Microsoft.NetCore.Analyzers.Publish.AvoidAssemblyLocationInSingleFile,
     Microsoft.CodeAnalysis.Testing.EmptyCodeFixProvider>;
 using static Analyzer.Utilities.MSBuildPropertyOptionNames;
-using static Microsoft.NetCore.Analyzers.Publish.AvoidAssemblyLocationInSingleFile;
```

Comment:
@mavasani

- IDE0005 (remove unnecessary imports) seems to be enabled for product code but disabled for tests. I ran a fix all so all files with unused usings got fixed (the number isn't large), I'm assuming this is okay?
- For this specific using in this test file, it's reported as unused. But removing it results in a compile-time error. I don't understand why there is a compile-time error after removing this `using static`. Note that this error occurs only in CI. Removing the using statement doesn't result in any errors on my machine.

Reply:
Can you please revert unrelated changes from this PR? Also the primary reason to disable this rule for test files is to enable us to have stub test files with required usings to help quick start writing tests. Let’s move this change and discussion to a separate PR

Reply:
Fixed